src/quo2/screens/lowest_price.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.screens.lowest-price
+  (:require [quo.design-system.colors :as colors]
+            [quo.react-native :as rn]
+            [re-frame.core :as re-frame]))
+
+
+(defn render-item
+  [{:keys [value1 value2]}]
+  
+  (let [{:keys [height width]} @(re-frame/subscribe [:dimensions/window])] 
+    [rn/view {:style {:display :flex
+                      :flex-direction :column
+                      :height "100%"
+                      :justify-content :center}}
+     [rn/view {:style {:overflow :hidden}}
+      [rn/view {:style {:border-width 2
+                        :border-color (:text-01 @colors/theme)
+                        :border-radius 1
+                        :margin -2
+                        :poisiton :absolute
```

Comment:
seems to probably be a typo here 
it would be `:position :absolute`

Reply:
Thanks Sid
Yeah it was a typo
Fixed